### PR TITLE
feat: LT-2704 - TAU Category 3 Products

### PR DESCRIPTION
**Pull request by iichr**
Add one new question to the add goods form, both from the product list and on a standard application:
- **Describe the purpose of the software/technology**
  - this is displayed only if the item category is in Group 3 (selected on the very first screen of the add a good flow) i.e. software or technology

### CODE REVIEW

File: templates/applications/goods/add-good-detail-summary.html [hunk 1]
```diff
@@ -83,6 +83,22 @@ <h1 class="govuk-heading-l">
...
+				</dt>
+				<dd class="govuk-summary-list__value">
+					{{ good.software_or_technology_details|default_na  }}
+				</dd>
+				<dd class="govuk-summary-list__actions">
+					{% if good.status.key == 'draft' %}
+						<a id="change-good-technology-software" class="govuk-link govuk-link--no-visited-state" href="{% url 'applications:good_software_technology' application_id good.id %}">Change</a>
+					{% endif %}
+				</dd>
+			</div>
```
[the remark this answers is not in the document]
Comment (iichr) [reply templates/applications/goods/add-good-detail-summary.html]: Added to all TAU detail change links on both summaries

File: applications/views/goods.py [hunk 2]
```diff
@@ -102,13 +102,17 @@ def init(self, request, **kwargs):
...
     def on_submission(self, request, **kwargs):
-        is_pv_graded = request.POST.get("is_pv_graded", "") == "yes"
-        self.forms = add_good_form_group(request, is_pv_graded, draft_pk=self.draft_pk)
+        copied_request = request.POST.copy()
+        is_pv_graded = copied_request.get("is_pv_graded", "") == "yes"
+        is_software_technology = copied_request.get("item_category") in ["group3_software", "group3_technology"]
+        self.forms = add_good_form_group(request, is_pv_graded, is_software_technology, draft_pk=self.draft_pk)
 
         if is_pv_graded:
+            # post on step 5 in both software/technology and group 1
```
Comment (Mark-Spiers-kainos) [inline applications/views/goods.py:111]: could you not just check if the form_pk is equal to the len of self.forms?

Comment (Mark-Spiers-kainos) [reply applications/views/goods.py:111]: Also comment and the if statement are quite different.

Comment (iichr) [reply applications/views/goods.py:111]: good spot, will amend as part of the next story as this flow will change

**Pull request merged**